dockerfiles/Dockerfile.use-installer

```diff
@@ -0,0 +1,76 @@
+ARG IMAGE="python:3-slim-buster"
+
+#---
+# Place anything that is common to both the build and execution environment in base 
+#
+FROM $IMAGE AS base
+
+RUN apt-get update -qq \
+ && DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends \
+    ca-certificates \
+    git \
+    libusb-1.0-0 \
+    make \
+    wget \
+ && apt-get autoclean && apt-get clean && apt-get -y autoremove \
+ && update-ca-certificates \
+ && rm -rf /var/lib/apt/lists
+
+# FROM base as builder 
+# Install necessary tools and source code and build from source here for ci/cd
+# If any build or test return a non-zero exit code, the docker build stops
+# with an error, which is nice for ci/cd like github actions.
+
+# FROM builder as test-builder
+# Run regression tests on binaries built from source here
+
+# FROM builder as installer
+# use COPY --from=builder to get the binaries from builder stage and
+# create an installer from those binaries built from source here
+
+FROM base as release-candidate
+# use COPY --from=builder to copy binaries into the right place here if built from source, 
+# or copy the installer with COPY --from=installer 
+# or use the pre-built released installer as we do here:
+RUN wget https://github.com/im-tomu/fomu-toolchain/releases/download/v1.5.6/fomu-toolchain-linux_x86_64-v1.5.6.tar.gz && \
```

Comment:
In this Dockerfiles are included in the repo, it is sensible to use the toolchain which is built in CI, not the latest stable. Using the latest stable makes running this workflow redundant most of the time.

This feels like something to be added in the documentation, for users willing to build custom images which include the toolcahin.